### PR TITLE
Move out audit code from libs package

### PR DESCRIPTION
**Pull request by evansmurithi**
fixes onaio/infrastructure#3923

### CODE REVIEW

File: audit/base.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ const (
 	Error
 )
 
-type Audit interface {
```
Comment (jasonrogena) [inline audit/base.go:24]: @evansmurithi since this is shared code, I think it should remain in the libs package.

Comment (jasonrogena) [reply audit/base.go:24]: Same applies for:
 - audit/base.go's test file
 - audit/port.go (including its tests)

Comment (evansmurithi) [reply audit/base.go:24]: @jasonrogena the code is only being used in the audit package

Comment (jasonrogena) [reply audit/base.go:24]: Sounds good.

**Review by jasonrogena — APPROVED** [review]

**Pull request merged**